src/ui/Controls/AdvancedTextBox.cs

```diff
@@ -23,7 +23,7 @@ public sealed class AdvancedTextBox : RichTextBox, IDoSpell
         private bool _checkRtfChange = true;
         private int _mouseMoveSelectionLength;
 
-        private bool IsLiveSpellCheckEnabled => Configuration.Settings.Tools.LiveSpellCheck;
+        private bool IsLiveSpellCheckEnabled => Configuration.Settings.Tools.LiveSpellCheck && Parent?.Name == "textBoxListViewText";
```

Comment:
Perhaps

`Parent?.Name == nameof(textBoxListViewText)`
or
`Parent == textBoxListViewText`

?

Reply:
It would be better, but isn't `textBoxListViewText` an instance in Main? How would this class see it? Am I missing something?

Reply:
Nope... would have been nice.

Reply:
Well, I can use a property in Main that returns the nameof it, what do you think?

Reply:
Could be a little better, so we avoid the string value... but I don't a see a super easy solution (I only see a public static property being set when creating the main form)

Reply:
This seems a bit sloppy https://github.com/OmrSi/subtitleedit/commit/d86ed449cd8ec79ba9aa17677cf75e06ab87afa5